src/FusedChannelData.ts

```diff
@@ -116,6 +132,45 @@ export default class FusedChannelData {
     this.maskTexture.dispose();
   }
 
+  private getShader(dtype: NumberType): ShaderMaterial {
+    switch (dtype) {
+      case "float32":
+        return this.fuseMaterialF;
+      case "uint8":
+      case "uint16":
+      case "uint32":
+        return this.fuseMaterialUI;
+      case "int8":
+      case "int16":
+      case "int32":
+        return this.fuseMaterialI;
+      default:
+        throw new Error("Unsupported data type for fuse shader");
+    }
+  }
+
+  private getMinMax(c: Channel): Vector2 {
```

Comment:
I'm glad you spotted this.  I sort of wanted to keep this function around in case we ever want to get those values into the shader but I suppose it would be easy enough to re-create again when it's needed.  I'll remove it